bears/rest/RstcheckBear.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class RstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck', '2.*')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file,
+            code_block_language_ignore: list):
+        """
+        A Bear for rstcheck. (https://pypi.python.org/pypi/rstcheck)
+        Rstcheck checks syntax of reStructuredText and code blocks
+        nested within it.
+        Supports bash, c, cpp, json, python and rst.
+
+        :param code_block_language_ignore:
+            comma seperated list for languages
+            of which code blocks should be
+            ignored.(bash,c,cpp,json,python,rst)
```

Comment:
what is this in the parentheses? Why is there no space before the paren? Are those options? Can you clarify this?